web/docs/DebuggingJest.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks'; 
+
+<Meta title="Documentation/Debugging Jest" />
+
+# Debugging
+
+## VS Code
+
+I found the following techniques helpful in understanding vscode debugging integration with jest test modules.  There are fancier vscode extensions (e.g. vscode-jest) that worked slick, but were a lot harder to understand in configuration -- this method uses a seperate launch configuration that you can execute from the debug console on the open test. Add this fragment to the launch.json inside the configurations property.
+
+It's worth noting that I struggled with the workspace configurations until I mounted the workspace at sub-project folder `FreeBoardgames.org/web` making web the root of the workspace. 
+
+[Article Describing Jest Debug using VSCode](https://www.basefactor.com/using-visual-studio-code-to-debug-jest-based-unit-tests)
+
+
+
+```JSON
+/** launch.json **/
```

Comment:
wouldn't it be easier to commit this to  the repo (without restricting it to a single game, or commiting one for  each game...) and just point out here what menus the developers need to go on VSCode to point  out?

Btw,  thanks for this, I use VSCode too :)

Reply:
@flamecoals not sure if you're awaiting some input?  My feeling is this type of config shouldn't be checked into code lines as it pollutes unnecessarily -- ever wrestled with a team wanting specific linters, code checks?  But this stuff should be close to project docs if project is to have a healthy community commitment.  I always appreciate devs insulating the team from their specific choices, but sharing them so we can develop better skills and/or tools.

Reply:
Sure, sounds good then. You can merge the docs :)